vendor/github.com/containers/common/libimage/push.go

```diff
@@ -36,6 +38,36 @@ func (r *Runtime) Push(ctx context.Context, source, destination string, options
 		return nil, err
 	}
 
+	// If specified to push all tags, look them up
+	if options.AllTags {
```

Comment:
You can not make changes in vendored directory.   You need to open a PR against containers/common to get this patch merged and then vendor it into Podman.

Reply:
Ah, I was wondering what that was all about. I'll get that done.